### ISSUE TITLE
Remove my `user@host` from the prompt

### ISSUE THREAD

**Issue by GalGreenfield** [agnoster/agnoster-zsh-theme#105]
I'm trying to remove the theme's `user@host` from the prompt.

 * I didn't modify the prompt via the `.bashrc` file.
 * I didn't modify the prompt via the `.zshrc` file.

In `agnoster.zsh-theme`, these are the things I tried to do (separately):
 * I uncommented the contents of the function `prompt_context`
 * I uncommented the whole of the `prompt_context` function
 * I uncommented the call for `prompt_context` in the `build_prompts` function

I might have tried a few combinations of the things above, I don't remember.

Nothing that I did worked - the prompt still kept displaying my `user@host`.

I verified that when Zsh is not running, the `user@host` is not displayed when I change PS1 to not display it in `.bashrc`.

Why is this happening and what can I do to solve it?

**Comment by Friz64** [comment]
user@host is displayed when your current user does not match the default user
https://github.com/robbyrussell/oh-my-zsh/issues/2033

tldr: add this to your .zshrc
```
DEFAULT_USER=`whoami` 
```

**Comment by apjanke** [comment]
Calling this solved.
